docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md

```diff
@@ -4,13 +4,18 @@ author: Harshit Gupta
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Understanding the flow of the AI-tools and what my project flow would be.
+- [x] Researching a method for chunking free text out from a stream of string.
+- [x] Implement a benchmark model and test it's results.
+- [x] Comparing the results with Jugalbandi API's method for text chunking.
+<!-- - [ ] Give the description about Milestone 4 -->
 
 ## Screenshots / Videos 
 
 ## Contributions
+- [Here's the link](https://colab.research.google.com/drive/1AgRUKep8juyyJcOik9kY1PFnkc8OCxhe?usp=sharing) to the google colab notebook used to create a benchmark and test it against Jugalbandi API
 
 ## Learnings
+- Looked out Mertics for Text segmentation and how to calculate them.
+- Learned more about text segmentation models.
```

Comment:
Hey, It would be better, if you can briefly explain all the learning.

Reply:
is it not personalized as of with the project scope? or are there any guidelines that I am not aware of?

Reply:
Can I add links of the blogs which I read instead for anyone to read? @Apoorvgarg-creator. I have added them into the tickets might as well add it here.

Reply:
Done. @Apoorvgarg-creator @wasup-yash

Reply:
Hey, @H4R5H1T-007 Sorry for not replying, I was busy ! Nice Updates

Reply:
> Hey, @H4R5H1T-007 Sorry for not replying, I was busy ! Nice Updates

So if everything is okay then please merge it now. @Apoorvgarg-creator @wasup-yash